content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Sensitive Data Masking"
+description: "How to manage sensitive data masking"
+tags: "admin policy management sensitive data masking"
+-->
+
+Take advantage of Contrast's data masking feature to limit risk to your organization and meet compliance requirements.  
+
+## How It Works
+
+Contrast's data masking feature protects sensitive data in your applications by preventing the Contrast agent from sending it to external systems: the Contrast UI, syslog or the security log. Data masking can also protect information by preventing the agent from sending it as part of Contrast vulnerability or attack reports. 
```

Comment:
The second sentence sounds redundant because of "can also protect", which could be confusing. Vulnerability and attack reports are the things being sent to "external systems", as listed in the first sentence. Maybe instead: "Sensitive data will be masked in vulnerability and attack reports."?

Also, the security log isn't really an external system, it's just a file. How about "transmitting or storing it externally" instead of "sending it to external systems"? This may be a bit nit-picky, so feel free to disregard.

Reply:
@jennymahmoudi I updated this paragraph to separate storing and transmitting into two separate sentences. If it didn't help (or made it worse), I'll give it another go.